io/include/pcl/io/pcd_io.h

```diff
@@ -252,6 +252,23 @@ namespace pcl
                             const Eigen::Quaternionf &orientation);
 
       /** \brief Generate the header of a BINARY_COMPRESSED PCD file format
+        * \param[out] os the stream into which to write the header
+        * \param[in] cloud the point cloud data message
+        * \param[in] origin the sensor acquisition origin
+        * \param[in] orientation the sensor acquisition orientation
+        *
+        * \return
+        *  * < 0 (-1) on error
+        *  * == 0 on success
+        */
+      int
```

Comment:
From the code this feels like it should return a Boolean instead of an integer. Was it to keep it consistent with all other methods returning integers as status?

Reply:
Yes (see https://github.com/PointCloudLibrary/pcl/pull/1869#discussion_r117749029)